tsdate/prior.py

```diff
@@ -947,7 +948,9 @@ def gamma_cdf(t_set, alpha, beta):
     return np.insert(t_set, 0, 0)
 
 
-def fill_priors(node_parameters, timepoints, ts, Ne, *, prior_distr, progress=False):
+def fill_priors(
+    node_parameters, timepoints, ts, population_size, *, prior_distr, progress=False
+):
```

Comment:
It would be useful to document the fact that the population_size parameter can now be an array (is this actually a strict requirement?)